webapp/src/main/webapp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "client",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
Is this right? Is the client version used/shown anywhere?

Reply:
Also wondering if there is a package-lock.json for RDW_Admin

Reply:
Yes we need to update the package-lock.json file too.

Reply:
I do not see where this is displayed on the client.

Reply:
do we want to give an actual `name` at this point as well?

Reply:
How about testApp1?

Reply:
I prefer `robTestApp14`.  Shows effort.